hubcap/package.py

```diff
@@ -41,6 +41,12 @@ def parse_pkgs(repo_dir):
         with open(repo_dir / Path("packages.yml"), "r") as stream:
             pkgs = yaml.safe_load(stream)
             return pkgs.get("packages", []) if pkgs else []
+    # new in v1.6
+    # a project can declare 'packages' in *either* packages.yml or dependencies.yml, not both
+    elif os.path.exists(repo_dir / "dependencies.yml"):
```

Comment:
I was going to note that this would silently fail if they did put something in both, but then I remembered that if they did that their project would fail to compile in the first place. So assuming people run their packages before submitting them to the hub, we can be a lil sloppier here and be confident that it'll be ok.